File: Aula04/desafios/desafio.cpp
#include <iostream>
using namespace std;

int main() {
    string nomes[3];
    int qtd[3];

    for(int i = 0; i < 3; i++){
        cout << "Produto: ";
        cin >> nomes[i];

        cout << "Quantidade: ";
        cin >> qtd[i];
    }

    cout << "\nRELATORIO:\n";

    for(int i = 0; i < 3; i++){
        if(qtd[i] < 5){
            cout << nomes[i] << " | " << qtd[i] << " [REPOSICAO NECESSARIA]\n";
        } else {
            cout << nomes[i] << " | " << qtd[i] << endl;
        }
    }

    return 0;
}

File: Aula05/desafios/desafio.py
def exibir_cabecalho():
    print("=== SORVETERIA DO DENER ===")

def verificar_estoque(qtd):
    if qtd < 5:
        return "[REPOSIÇÃO NECESSÁRIA]"
    return ""

nomes = []
qtds = []

exibir_cabecalho()

for i in range(3):
    nomes.append(input("Produto: "))
    qtds.append(int(input("Quantidade: ")))

print("\nRELATÓRIO:")

for i in range(3):
    aviso = verificar_estoque(qtds[i])
    print(f"{nomes[i]} | {qtds[i]} {aviso}")